src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/prometheus/monitoring-list/monitoring-list.component.html

```diff
@@ -1,7 +1,7 @@
-<tabset #tabs>
-  <tab id="active-alerts"
+<tabset>
+  <tab cdTabActivateByFragment
+       fragment="active-alerts"
        heading="Active Alerts"
```

Comment:
> Sorry for the late review.
> 
> We currently have a different approach for this problem in the block images pages and I think I like it better.
> 
> Few things to consider:
> 
>     * Only the active tab should be loaded, to reduce memory consumption and requests made
>       
>       * RBD implementation this happens automatically, since only 1 component is loaded
>       * Should be easy to implement in this solution
> 
>     * RBD has a cleaner code in the "index" component, like  `rbd-tabs` and  in this case would be `monitoring-list`
> 
>     * RBD doesn't have 2 fragments in the URL
>       
>       * although this seems to work, it looks strange to me
> 
>     * `rbd-tabs.component.html` could be improved further by refactoring the code used in [active] and (selectTab) by declaring the url only once.
> 
>     * In RBD we can find out all the possible URL of ceph dashboard by going through routing declarations, currently don't think it matter, but could be used in a future for something like a sitemap
> 
>     * In RBD changing tab also updates the breadcrumb
> 
> 
> Independently of which solution we decide on, I think we should refactor all components similar to these ones to use the same.

I will refactor the monitoring-list like it is done in rbd using a `monitoring-tabs` component. I think this will be the better approach.